src/provider/model/Model.ts

```diff
@@ -236,6 +176,33 @@ export class Model {
         }
     }
 
+    /**
+     * Registers an appWindow in the model
+     * @param appInfo Application info, either from the app directory, or 'crafted' for a non-registered app
+     * @param identity Window identity
+     * @param isInAppDirectory boolean indicating whether the app is registered in the app directory
+     */
+    private registerWindow(appInfo: Application, identity: Identity, isInAppDirectory: boolean): AppWindow {
+        const appWindow = this._environment.wrapApplication(appInfo, identity, this._channelsById[DEFAULT_CHANNEL_ID]);
+        appWindow.channel = this._channelsById[DEFAULT_CHANNEL_ID];
```

Comment:
Seems this assignment happens within `wrapApplication` already?

Also, when wrapping an application is `channel` ever anything other than `DEFAULT_CHANNEL_ID`? Didn't see any instances, but the interface makes searching a little fiddly.

Reply:
Changed